src/main.zig

```diff
@@ -2581,6 +2587,12 @@ fn buildOutputType(
             clang_argv.appendAssumeCapacity(include_dir);
         }
 
+        try clang_argv.ensureUnusedCapacity(framework_dirs.items.len * 2);
+        for (framework_dirs.items) |framework_dir| {
+            clang_argv.appendAssumeCapacity("-F");
```

Comment:
Should this not be `-iframework`? After all, clang only cares about include paths.

Reply:
In the `clang` documentation, `-F` is to `add directory to framework include search path`. However, `-iframework` is to `add directory to SYSTEM framework search path`. I believe the original issue was to ask for the ability to add a custom framework (e.g. the SDL2 framework) to the search path. Would we want to add it to the system framework search path?